web/src/main/kotlin/no/nav/su/se/bakover/web/Application.kt

```diff
@@ -110,44 +105,34 @@ internal fun Application.susebakover(
                 jmsConfig,
                 clock = clock,
             ).build(applicationConfig),
-    services: Services =
-        with(
-            if (applicationConfig.runtimeEnvironment == ApplicationConfig.RuntimeEnvironment.Nais)
-                ProdServiceBuilder
-            else
-                StubServiceBuilder
-        ) {
-            build(
-                databaseRepos = databaseRepos,
-                clients = clients,
-                behandlingMetrics = behandlingMetrics,
-                søknadMetrics = søknadMetrics,
-                clock = clock,
-                unleash = DefaultUnleash(
-                    UnleashConfig.builder()
-                        .appName(applicationConfig.unleash.appName)
-                        .instanceId(applicationConfig.unleash.appName)
-                        .unleashAPI(applicationConfig.unleash.unleashUrl)
-                        .build(),
-                    IsNotProdStrategy(applicationConfig.naisCluster == ApplicationConfig.NaisCluster.Prod)
-                )
+    services: Services = if (applicationConfig.runtimeEnvironment == ApplicationConfig.RuntimeEnvironment.Nais) {
+        ProdServiceBuilder.build(
```

Comment:
Siden den samme servicebuilderen brukes uansett; kunne den ha hett bare `ServiceBuilder`? Eller er det kanskje noe annet for testing?